oasislmf/execution/bash.py

```diff
@@ -1046,11 +1046,13 @@ def genbash(
     print_command(filename, '')
 
     if fmpy:
-        print_command(
-            filename, f'{get_fmcmd(fmpy)} -a{il_alloc_rule} --create-financial-structure-files'
-        )
-        for i in range(1, num_reinsurance_iterations + 1):
-            print_command(filename, f'{get_fmcmd(fmpy)} -a{ri_alloc_rule} --create-financial-structure-files -p RI_{i}')
+        if il_output:
```

Comment:
you will need this if:
il_output or ri_output is true because if you have ri and not il you still need to create the financial file to compute il.